server/app/models/host_node.rb

```diff
@@ -31,6 +31,10 @@ class HostNode
   field :agent_version, type: String
   field :docker_version, type: String
   field :connected_at, type: Time
+  field :disconnected_at, type: Time
+  field :updated, type: Boolean, default: false # true => node sent /nodes/update after connecting; false => node attributes may be out of date even if connected
+  field :connection_error_code, type: Integer
```

Comment:
Should we combine connection error code & message to single field (hash or embedded object)?